decidim-forms/app/forms/decidim/forms/admin/questionnaire_form.rb

```diff
@@ -11,16 +11,7 @@ class QuestionnaireForm < Decidim::Form
         translatable_attribute :description, String
         translatable_attribute :tos, String
 
-        attribute :published_at, Decidim::Attributes::TimeWithZone
```

Comment:
I see that this has moved to the "QuestionsForm", can you tell me the reason?

Reply:
I got it now, its not been used.